uw-frame-static/index.html

```diff
@@ -15,11 +15,20 @@
   <link ng-href="css/themes/uw-{{portal.theme.name}}.css" rel="stylesheet" type="text/css"/>
   <link href="my-app/my-app.css" rel="stylesheet" type="text/css"/>
   <link rel="shortcut icon" href="bower_components/uw-ui-toolkit/dist/img/favicon.ico" type="image/x-icon"/>
+  <link rel="import" href="static.html" />
```

Comment:
What browsers support this? Is [this](https://developer.mozilla.org/en-US/docs/Web/Web_Components/HTML_Imports)

> Firefox will not ship HTML Imports

worrying?